README.md

```diff
@@ -6,8 +6,7 @@ See documentation [here](doc/00-overview.md)
 
 python:
     reason: 'make sure python flag exists to load config in python.md'
-debug-output-folder: $(output-folder)/_az_debug
-az-output-folder: $(output-folder)
+debug-output-folder: $(az-output-folder)/_az_debug

```

Comment:
>az-output-folder [](start = 23, length = 16)

the az-output-folder is empty in this case, isn't it?

Reply:
no autorest will read the configuration from the readme.az.md

Reply:
i see, then we should set output-folder=az-output-folder, right?

---
In reply to: [407913102](https://github.com/Azure/autorest.az/pull/302#discussion_r407913102) [](ancestors = 407913102)

Reply:
emmm, you mean don't change the python-sdk-output-folder in readme.az.md?